website/client/components/appFooter.vue

```diff
@@ -53,7 +53,7 @@
             h3 Developers
             ul
               li
-                a(href='/apidoc', target='_blank') {{ $t('APIv3') }}
+               router-link(to='/apidoc', target='_blank') {{ $t('APIv3') }}
```

Comment:
links with `target="_blank"` shouldn't be changed

Reply:
Fixed in the next commit